src/components/Panel.scss

```diff
@@ -35,36 +35,121 @@ $Panel-header-text-color: $cu-info !default;
   box-sizing: border-box;
 }
 
+// 1. Panel body has no padding by default. Padding is set on rows and cells,
+//    instead. Can be overriden in markup with spacing utilities as necessary.
+// 2. Override some styles applied to the base <table> element, so that
+//   .Panel-body can be applied to a <table>.
+
+.Panel-body {
+  padding: 0; // 1
+  border: 0; // 2
+  margin-bottom: 0; // 2
+  width: 100%; // 2
+}
+
+// Hide overflowing panel corners by setting border radius on corner children
+
+.Panel-body:first-child .Panel-row:first-child .Panel-cell:first-child {
+  border-top-left-radius: $border-radius-large;
+}
+
+.Panel-body:first-child .Panel-row:first-child .Panel-cell:last-child {
+  border-top-right-radius: $border-radius-large;
+}
+
+.Panel-body:last-child .Panel-row:last-child .Panel-cell:first-child {
+  border-bottom-left-radius: $border-radius-large;
+}
+
+.Panel-body:last-child .Panel-row:last-child .Panel-cell:last-child {
+  border-bottom-right-radius: $border-radius-large;
+}
+
 .Panel-header,
-.Panel-body,
-.Panel-footer {
+.Panel-footer,
+.Panel-row,
+.Panel-cell {
   padding: $su-default;
 }
 
 .Panel-header {
+  border-bottom: $border-width-small solid $cu-divider;
+}
+
+.Panel-title {
   color: $Panel-header-text-color;
   font-family: $base-font-family;
   font-size: $tu-base-fontSize;
   font-weight: $tu-bold-fontWeight;
-  border-bottom: $border-width-small solid $cu-divider;
+  padding: 0;
+  margin: 0;
 }
 
-.Panel-row {
-  padding: $su-small 0;
+.Panel-row:not(:last-child) {
   border-bottom: $border-width-small solid $cu-divider--light;
+}
 
-  &:first-child {
-    padding-top: 0;
-  }
-
-  &:last-child {
-    padding-bottom: 0;
-    border-bottom: 0;
-  }
+// Override default <table> cell styles for when table markup is used
+.Panel-cell {
+  border-top: 0;
+  border-bottom: 0;
+}
 
+.Panel-cell--header {
+  background-color: $cu-background--light;
```

Comment:
Same here... just anywhere anywhere it's currently used here... Since background--light === background, $cu-middleground is probably the one you were looking for.